cli/replicaset/cmd/upgrade.go

```diff
@@ -0,0 +1,272 @@
+package replicasetcmd
+
+import (
+	_ "embed"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/mitchellh/mapstructure"
+	"github.com/tarantool/tt/cli/connector"
+	"github.com/tarantool/tt/cli/replicaset"
+	"github.com/tarantool/tt/cli/running"
+)
+
+var (
+	// List of replicaset names specified by the user for the upgrade.
+	ChosenReplicasetAliases []string
+	// Timeout period (in seconds) for waiting on LSN synchronization.
+	LsnTimeout int
+)
```

Comment:
Please, rewrite the code to avoid using of the global variables. This creates a lot of non-obvious relationships in the code that are more difficult to maintain and test. It's better to create a some type and pass objects of the type into calls.